site/content/docs/browser-checks/login-scenarios.md

```diff
@@ -83,121 +103,88 @@ redirects involved. Also, many providers make their login pages "bot resistant"
+{{< tab "Javascript" >}}
+```js
+const { test } = require('@playwright/test')
 
-const browser = await puppeteer.launch()
-const page = await browser.newPage()
+test('Login into Checkly with Google social login', async ({ page }) => {
+    await page.goto('https://app.checklyhq.com/')
 
-await page.setViewport({ width: 1280, height: 800 })
-await page.goto('https://checklyhq.com/login')
+    // Click Google login
+    await page.getByText('Sign in with Google').click()
 
-const navigationPromise = page.waitForNavigation()
-await page.goto('https://app.checklyhq.com/')
+    // Provide email address and click 'next' button
+    await page.locator('input[type="email"]').type(process.env.GOOGLE_USERNAME)
+    await page.getByRole('button', { name: 'Next' }).click()
 
-// Click Google login and wait for redirect
-await page.click('a[data-provider="google-oauth2"]')
-await navigationPromise
+    // Provide password, click 'sign in' button
+    await page.locator('input[type="password"]').type(process.env.GOOGLE_PASSWORD)
+    await page.getByRole('button', { name: 'Sign in' }).click()
 
-// provide email address and click next
-await page.waitForSelector('input[type="email"]')
-await page.type('input[type="email"]', process.env.GOOGLE_USER)
-await page.click('#identifierNext')
-
-// provide password, click next and wait for redirect back to Checkly
-await page.waitForSelector('input[type="password"]', { visible: true })
-await page.type('input[type="password"]', process.env.GOOGLE_PWD)
-
-const navigationPromise2 = page.waitForNavigation()
-await page.waitForSelector('#passwordNext', { visible: true })
-await page.click('#passwordNext')
-await navigationPromise2
-
-await browser.close()
+    // Verify successful login
+    await page.getByTestId('home-dashboard-table').toBeVisible()
+})
 ```
 {{< /tab >}}
 {{< /tabs >}}
 
 Note the following:
 
-- We create a `navigationPromise` ahead of time and wait for this promise to resolve as we navigate from domain to domain.
 - We of course store our credentials in environment variables.
-- We use the `state: 'visible'`/`visible: true`(for Playwright and Puppeteer, respectively) option when waiting for the buttons and input fields to appear. The script fails otherwise.
+- Playwright Test's locators will wait for your elements to be visible until the test times out (which will make the check fail).
 
 ## Password-protected websites
 
-In certain cases, for example with [Vercel password-protected deployments](https://vercel.com/blog/protecting-deployments), websites might require a password to be entered before the target page is made available. Much like login cases, this can be solved directly using Puppeteer or Playwright:
+In certain cases, for example with [Vercel password-protected deployments](https://vercel.com/blog/protecting-deployments), websites might require a password to be entered before the target page is made available. Much like login cases, this can be solved directly using Playwright Test:
```

Comment:
This section seems like a duplication of the first scenario. Should we combine them?

Reply:
Yep, good idea. Seems redundant.